versions/v6/static/scss/_footer.scss

```diff
@@ -50,6 +50,8 @@
   }
 
   .story-callout {
+    @include make-col(5);
```

Comment:
It might make more sense to adjust `.story-callout` widths in scss/components/_story.list.scss instead of as a footer-specific override here to ensure the top nav pulldown also shows 2.5 stories at a time on mobile.